packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -496,6 +503,18 @@ export class Markets {
       });
     }
 
+    if (params.templateFilter) {
+      if (params.templateFilter === TemplateFilters.templateOnly) {
+        request.selector = Object.assign(request.selector, {
+          isTemplate: { $eq: true },
+        });
+      } else if (params.templateFilter === TemplateFilters.customOnly) {
+        request.selector = Object.assign(request.selector, {
+          isTemplate: { $eq: false },
+        });
+      }
+    }
+
```

Comment:
Github won't let me do a suggestion for more than one line... anyway I'd shorten this by doing something like

```
if (params.templateFilter && params.templateFilter !== TemplateFilters.all) {
  request.selector.isTemplate = { $eq: params.templateFilter === TemplateFilters.templateOnly }
}
```

Reply:
I want to only process `templateOnly` and `customOnly` and ignore any other values.

Reply:
How about this?

```
if (params.templateFilter === TemplateFilters.templateOnly) {
  request.selector.isTemplate = { $eq: true };
} else if (params.templateFilter === TemplateFilters.customOnly) {
  request.selector.isTemplate = { $eq: false };
}
```

Reply:
I can slim down the code.

Reply:
the suggestion above gives linting errors because request.selector.isTemplate doesn't exist on object. I don't want to add isTemplate there if 'all' is selected need to leave it off.